plugins/custom-dashboard/README.md

```diff
@@ -0,0 +1,35 @@
+# Custom Dashboard Plugin
```

Comment:
Can we rename `Custom Dashboard` to `Dashboard` for conciseness throughout here? This is *the* dashboarding tool 😄